### PR TITLE
Bungee sync msg + pom.xml edit

### PR DESCRIPTION
**Pull request by xPiston**
Addition of sending messages between servers.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -238,21 +238,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```
Comment (EpicEricEE) [inline pom.xml:241]: Why did you remove this?

File: src/main/java/de/epiceric/shopchest/ShopChest.java [hunk 2]
```diff
@@ -99,6 +99,7 @@
     private ShopUpdater updater;
     private ExecutorService shopCreationThreadPool;
 
+
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/ShopChest.java:102]: unnecessary empty line

File: src/main/java/de/epiceric/shopchest/config/Config.java [hunk 3]
```diff
@@ -219,6 +219,11 @@
      **/
     public static boolean enableVendorMessages;
 
+    /**
+     * Whether the vendor of the shop should get messages about buys and sells
+     **/
+    public static boolean enableVendorBungeeMessages;
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/config/Config.java:225]: add the property to the `config.yml` file

File: src/main/java/de/epiceric/shopchest/shop/Shop.java [hunk 4]
```diff
@@ -473,4 +476,20 @@ public InventoryHolder getInventoryHolder() {
         return null;
     }
 
+    public void sendbungeeMessage(String player, String message) {
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/shop/Shop.java:479]: can this be a private method in the ShopInteractListener? This doesn't seem like it's related to the shop

File: src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
+package de.epiceric.shopchest.utils;
+
+import de.epiceric.shopchest.ShopChest;
+import org.bukkit.entity.Player;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import java.io.ByteArrayOutputStream;
+
+public class PluginMessageTask extends BukkitRunnable {
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java:9]: Is this class really needed? I feel like you could have a simple 
```java
plugin.getScheduler().runTaskAsynchronously(() -> player.sendPluginMessage(plugin, "BungeeCord", message));
```
in the `sendBungeeMessage()` method and use the player who bought/sold as the online player.

File: pom.xml (hunk 1 above)
Comment (xPiston) [reply pom.xml]: I suppressed it because it spanked me whatever error. I just put it back in my last commit: https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa

File: src/main/java/de/epiceric/shopchest/ShopChest.java (hunk 2 above)
Comment (xPiston) [reply src/main/java/de/epiceric/shopchest/ShopChest.java]: Fix in my last commit : https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa

File: src/main/java/de/epiceric/shopchest/config/Config.java (hunk 3 above)
Comment (xPiston) [reply src/main/java/de/epiceric/shopchest/config/Config.java:225]: I just added it to my commit: https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa

File: src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java (hunk 5 above)
Comment (xPiston) [reply src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java:9]: https://github.com/xPiston/ShopChest/commit/8ec8a63e4b8151de08dd18ba1ec1b7c140b0f0aa

File: src/main/java/de/epiceric/shopchest/listeners/ShopInteractListener.java [hunk 6]
```diff
@@ -20,11 +20,7 @@
 import de.epiceric.shopchest.shop.ShopProduct;
 import de.epiceric.shopchest.shop.Shop.ShopType;
 import de.epiceric.shopchest.sql.Database;
-import de.epiceric.shopchest.utils.ClickType;
-import de.epiceric.shopchest.utils.ItemUtils;
-import de.epiceric.shopchest.utils.Permissions;
-import de.epiceric.shopchest.utils.ShopUtils;
-import de.epiceric.shopchest.utils.Utils;
+import de.epiceric.shopchest.utils.*;
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/listeners/ShopInteractListener.java:23]: Could you try to avoid wildcard imports please?

File: src/main/java/de/epiceric/shopchest/listeners/ShopInteractListener.java [hunk 7]
```diff
@@ -1157,4 +1170,19 @@ private boolean removeFromInventory(Inventory inventory, ShopProduct product) {
         return (removed == amount);
     }
 
+    public void sendbungeeMessage(String player, String message) {
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/listeners/ShopInteractListener.java:1173]: `sendBungeeMessage` please with an upper case `B`

File: src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java (hunk 5 above)
Comment (EpicEricEE) [reply src/main/java/de/epiceric/shopchest/utils/PluginMessageTask.java:9]: I mean, you could remove the whole `PluginMessageTask` class and use this as the `sendBungeeMessage()`:
```java
public void sendBungeeMessage(String player, String message) {
    try {
        ByteArrayOutputStream b = new ByteArrayOutputStream();
        DataOutputStream out = new DataOutputStream(b);

        out.writeUTF("Message");
        out.writeUTF(player);
        out.writeUTF(message);

        if (!plugin.getServer().getOnlinePlayers().isEmpty()) {
            plugin.getServer().getScheduler().runTaskAsynchronously(plugin, () -> {
                Player player = plugin.getServer().getOnlinePlayers().iterator().next();
                player.sendPluginMessage(plugin, "BungeeCord", b.toByteArray());
            });
        }
    } catch (Exception e) {
        plugin.debug("Failed to send bungee message");
        plugin.debug(e);
        plugin.getLogger().warn("Failed to send BungeeCord message")
    }
}
```

File: src/main/java/de/epiceric/shopchest/shop/Shop.java [hunk 8]
```diff
@@ -25,6 +26,8 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.scheduler.BukkitRunnable;
 
+import java.io.ByteArrayOutputStream;
```
Comment (EpicEricEE) [inline src/main/java/de/epiceric/shopchest/shop/Shop.java:29]: Also, remove these imports
